ros-kinetic/gscam/gscam-1.0.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Open Source Robotics Foundation
+# Distributed under the terms of the BSD license
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_5,3_6} )
+
+inherit ros-cmake
+
+DESCRIPTION="A ROS camera driver that uses gstreamer to connect to\
+	devices such as [...]"
+HOMEPAGE="https://wiki.ros.org"
+SRC_URI="https://github.com/ros-drivers-gbp/${PN}-release/archive/release/melodic/${PN}/1.0.1-0.tar.gz -> ${PN}-melodic-release-${PV}.tar.gz"
```

Comment:
@allenh1 I think this is why you see a mention of melodic. It should be harmless, as it's just a name of the file.